### PR TITLE
Feat/cart

### PR DESCRIPTION
**Pull request by SimranjitKaurMaan**
1. Added Add to Cart Functionality
2. Remove from Cart
3. Increment qty/decrement qty
add_to_cart_demo

### CODE REVIEW

File: src/backend/controllers/CartController.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ export const addItemToCartHandler = function (schema, request) {
     }
     const userCart = schema.users.findBy({ _id: userId }).cart;
     const { product } = JSON.parse(request.requestBody);
-    console.log(`product in addItemToCartHandler: ${product}`)
+    console.log(`product in addItemToCartHandler: ${JSON.stringify(product)}`)
```
Comment (kjais1720) [inline src/backend/controllers/CartController.js]: remove the log

File: src/components/Cart.js [hunk 2]
```diff
@@ -1,47 +1,67 @@
...
+    const totalDiscount = totalPrice*0.2;
+    const deliveryCharges = totalPrice*0.05;
+    const amountToPay = totalPrice - totalDiscount + deliveryCharges;
 
-    const DeleteFromCartHandler = async (event) => {
-        const selectedProduct = cartProducts.find(product => product._id === event.target.id);
-        const response = await deleteFromCart(selectedProduct._id);
-        console.log(response);
-     }
+    console.log(`state in cart: ${JSON.stringify(state)}`);
```
Comment (kjais1720) [inline src/components/Cart.js]: here as well

File: src/components/home /Trending.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ export const Trending = () => {
     useEffect(() => {
         (async () => {
         const products = await fetchProducts();
-        console.log("inside trending ",JSON.stringify(products));
+        //console.log("inside trending ",JSON.stringify(products));
```
Comment (kjais1720) [inline src/components/home /Trending.js]: remove the comment

**Review by kjais1720 — APPROVED** [review]
Everything looks good, I especially liked the handling of images for different screen widths

**Review by raman1999 — APPROVED** [review]
code looks clean to me, just remove unnecessary comments and consoles  inn your code

**Pull request merged**